.github/workflows/ci.yaml

```diff
@@ -42,6 +42,12 @@ jobs:
           - windows-latest
 
     steps:
+      - name: Configure git
+        working-directory: ~
+        run: |
+          git config --global core.autocrlf input
+          git config --global core.eol lf
+
```

Comment:
Had for me to comment much because I don't really understand why this was done! I know it has to do with newlines on Win vs Unix, but I don't understand the exact details. I think it is best if you just provide a comment here explaining why this was done, and that is it.